app-devel/intel-oneapi-basekit/autobuild/templates

```diff
@@ -0,0 +1,8 @@
+Template: intel-oneapi-basekit/setvars_postinst
+Type: note
+Description: Set up environment for Intel oneAPI toolkit
+ To set binary/library paths properly and use Intel oneAPI toolkit,
```

Comment:
The user environment for using Intel oneAPI Toolkit is set up on-demand. To use the Intel oneAPI Toolkit, please use the following command as the current user: